frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -451,6 +453,9 @@
                     formData.append("method_description", vm.methodDesc);
                     formData.append("project_url", vm.projectUrl);
                     formData.append("publication_url", vm.publicationUrl);
+                    formData.append("submission_meta_attributes", vm.submission_meta_attributes);
```

Comment:
@Ram81 This is where I'm storing the attributes. Do I need to convert it into something before appending here. It should be stored as JSONField on the backend.